count_down/web/tute_countdown.dart

```diff
@@ -42,19 +44,19 @@ class CountDownComponent extends PolymerElement {
     errorMsg = '';
     appObject.clear();
   }
-   
+
   // Life-cycle bizness
-  void attached() {
+  attached() async {
     super.attached();
-    appObject.start()
-      .catchError((e) {
-        ($['addbutton'] as ButtonElement).disabled = true;
-        ($['clearbutton'] as ButtonElement).disabled = true;
-
-        errorMsg = e.toString();
-      });
+    try {
+      appObject.start(); // Returns a future.
```

Comment:
Drive-by comments: you should use 'await appObject.start();' otherwise the try-catch will have no effect.